modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -642,9 +642,10 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
       case Field(name, _, tpe, required, hints) =>
         val line = line"$tpe"
         val tpeAndDefault = if (required) {
-          val maybeDefault = hints
+          val maybeRenderedDefault = hints
             .collectFirst { case d @ Hint.Default(_) => d }
             .map(renderDefault)
```

Comment:
not sure if it would have been better, but an alternative would have been to add `.filterNot(_ => noDefault)`

Reply:
I like it, updated 👍